docs/source/en/quantization/bitsandbytes.md

```diff
@@ -64,7 +46,7 @@ model_8bit = AutoModelForCausalLM.from_pretrained(
 )
 ```
 
-By default, all the other modules such as `torch.nn.LayerNorm` are converted to `torch.float16`. You can change the data type of these modules with the `torch_dtype` parameter if you want. Setting `torch_dtype="auto"` loads the model in the data type defined in a model's `config.json` file.
+By default, all other modules such as [torch.nn.LayerNorm](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html) are converted to `torch.float16`. You can change the data type of these modules with the `torch_dtype` parameter.. Setting `torch_dtype="auto"` loads the model in the data type defined in a model's `config.json` file.
```

Comment:
I'm not sure this is accurate. I would think it would follow whatever the torch default dtype is unless `torch_dtype` is set. The bitsandbytes integration does not do anything to a module is not `nn.Linear`.

Reply:
Clarified that it is set to just the default torch dtype unless explicitly set with `torch_dtype` parameter!